src/routes/NotFound.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
```

Comment:
Since this is a component, I think this file should live in components/common folder. But, probably still work re-visiting organization as a team.